src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -105,7 +105,7 @@ case class CsvRelation protected[spark] (
     tokenRdd(schemaFields.map(_.name)).flatMap { tokens =>
 
       if (dropMalformed && schemaFields.length != tokens.size) {
-        logger.warn(s"Dropping malformed line: ${tokens.mkString(",")}")
+        logger.warn(s"Dropping malformed line (wrong length): ${tokens.mkString(",")}")
```

Comment:
Personal opinion: likewise, shouldn't we maybe need a more detail message rather than just printing `wrong length`? e.g.` Expected token length is 5 but the parsed token length was 4`.